spec/features/user_sign_ups_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.feature "User Sign Up", type: :feature do
+    scenario 'successfully' do
+        visit '/users/sign_up'
+        expect(page).to have_css('h2', text: 'New User Sign Up')
+        fill_in 'First name', with: 'Ian'
+        fill_in 'Last name', with: 'Roberts'
+        fill_in 'Email', with: 'ianroberts131@gmail.com'
+        fill_in('Password', with: "password", exact: true)
+        fill_in('Password confirmation', with: 'password', exact: true)
+        click_button 'Sign up'
+        
+        expect(page).to have_text "Welcome"
+    end
```

Comment:
@ianroberts131 Good work adding test coverage for user sign up. A couple of minor things to consider adding:

- Line 14: The expectation that checks the user signed up completed successfully is quite general (checking the text "Welcome" is present isn't a clear enough signal that the sign up completed). A clearer signal that confirms the user signed up successfully would be to check for the flash notice text that Devise sets which is usually something like "You have signed up successfully".
- Check the indentation setting in the editor. This looks like 4 spaces. Ensure its 2 soft-spaces as this is the usual convention when writing Ruby.